content/en/product_analytics/session_replay/heatmaps.md

```diff
@@ -82,21 +82,35 @@ The panel to the left of the Scroll map provides high-level insights with direct
 
 {{< img src="real_user_monitoring/session_replay/heatmaps/heatmaps-minimap.png" alt="A screenshot of the queries for scroll data insights" style="width:50%;">}}
 
-## Snapshots
+## Screenshots
 
-A snapshot is the state of a Session Replay at a particular point in time. Changing the snapshot shows different results, depending on the snapshot selected. You can use the **Change Snapshot** button to select a particular snapshot in a replay for your heatmap.
+A screenshot is the state of a view at a particular point in time. Changing the screenshot shows different results, depending on the screenshot selected. You can also save screenshots so that everyone in your organization is analyzing the same view state. 
```

Comment:
```suggestion
A screenshot is the state of a view at a particular point in time. Changing the screenshot shows different results, depending on the screenshot selected. You can also save screenshots so that everyone in your organization can analyze the same view state.
```